admin-ui/plugins/auth-server/components/Agama/AgamaAliasListPage.js

```diff
@@ -61,14 +71,14 @@ function AliasesListPage() {
     dispatch(getJsonConfig({ action: {} }))
   }, [dispatch])
 
-  // Build actions based on permissions
   useEffect(() => {
+    const t = (s) => s
```

Comment:
Defining a local identity function that shadows the internationalization translation function is bad because it violates internationalization best practices by creating a no-op function named 't' that overrides the actual translation utility, which prevents UI labels from being localized and creates confusing code.